### ISSUE TITLE
BUG on more two calls at  the same time

### ISSUE THREAD

**Issue by incomingflyingbrick** [SimpleMobileTools/Simple-Dialer#290]
Hi, I am using Simple Dialer. While I am calling someone , another call coming which I answered. I hang up now, and I can still talk to the first person. But When i hang out the first call, it won't hang up. 

Another question is Can I record calls with this app, if yes, how do I do it?

**Comment by tibbi** [comment]
nope, call recording isnt currently supported on Android

**Comment by kamaravichow** [comment]
@tibbi  I think you can add it using accessibility service for android versions over android 11.

**Comment by tibbi** [comment]
not really, you cant. It might work just on a couple of devices and it would bring a lot of bad reviews from the others

**Comment by Jumbo125** [comment]
is there any solution about "hang up first call"?

**Comment by Jumbo125** [comment]
today it happened again. Is there a solution?  i read the posts, but i didn't found it. Is it my mistake? 
Thank you for help

**Comment by Aga-C** [comment]
There is no fix right now. I tried to find how to fix it, but unfortunately I haven't found a solution. Maybe there are some devs who know how to fix it, and may help with that.

**Comment by Jumbo125** [comment]
okay. Thank you

**Comment by Jumbo125** [comment]
Oh, do you know any trick to cancel the call without shut off the phone?
